clients/python-wrapper/tests/integration/test_managers.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from tests.integration.conftest import _setup_repo, get_storage_namespace
+
+
+@pytest.fixture(name="setup_repo_with_branches_and_tags", scope="session")
+def fixture_setup_repo_with_branches_and_tags():
+    clt, repo = _setup_repo(get_storage_namespace("branches-and-tags"),
+                            "branches-and-tags",
+                            "main")
+    for i in range(10):
+        for j in range(100):
+            b = repo.branch(f"branches{i:02d}-{j:03d}").create("main")
+            repo.tag(tag_id=f"tags{i:02d}-{j:03d}").create(b)
+
+    return clt, repo
+
+
+@pytest.mark.parametrize("attr", ("branches", "tags"))
+def test_branch_manager(setup_repo_with_branches_and_tags, attr):
```

Comment:
Testing X manager?
Still think we can have one test method for each or have two entry points and call the same test method

Reply:
Fixed